ChangeLog

```diff
@@ -34,6 +34,10 @@ Release date: TBA
 
   Close #3147
 
+ * Add new good-names-rgx and bad-names-rgx to enable white-/blacklisting of regular expressions
```

Comment:
Can you move this under the 2.5 section?

Reply:
done